index.d.ts

```diff
@@ -239,6 +239,22 @@ declare namespace Obyte {
         callback?: (err: null | string, result: string) => void,
       ): Promise<object>;
 
+      getAasByBaseAss(
+        params: {
+          base_aa: object;
```

Comment:
The type seem wrong, this should be a string and be optional
```suggestion
          base_aa?: string;
```
Same for `base_aa` and `aa`, `aas` on `getAaResponses`